lib/express-rate-limit.js

```diff
@@ -85,6 +86,16 @@ function RateLimit(options) {
               );
             }
           }
+          if (options.draft_polli_ratelimit_headers && !res.headersSent) {
```

Comment:
@nfriedly WDYT to set the header field names in the configuration?
This will:

- avoid duplicate code in L78-79 and L90-91
- concentrate changes to L82
- provide more flexibility

Reply:
Those sections aren't duplicates, L78-79 has the `X-` prefix and L90-91 doesn't.

I like it the way it is now because:

1) it's backwards-compatible
2) it allows for users to choose one or both styles of headers (with or without `X-`)

I'm anticipating a future semver major release that deprecates and removes the `X-` headers; that should provide the simplification you're looking for. But, for now I think it's best to support both varieties and ship this as a semver minor.

Reply:
Ok, let's support both for now! It seems a good choice!